src/main/assembly/dist/docs/sword-v1-packaging.html

```diff
@@ -0,0 +1,564 @@
+<!DOCTYPE html>
+<head lang="en">
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title lang="en">Sword Packaging for EASY Sword-v1</title>
+    <style>
+    body {
+        max-width: 1200px;
+        margin: auto;
+        padding: 1em;
+        font-family: "Open Sans",Arial,Helvetica,sans-serif;
+    }
+    article {
+        border: solid 1px;
+        margin-bottom: 1em;
+        margin-right: 1em;
+        border-radius: 0.5em;
+        padding: 1em;
+    }
+    #mapping, #helpIndex {
+        display: inline-block;
+        vertical-align: top;
+    }
+    th {
+        text-align: left;
+    }
+    hr {
+        display: none;
+    }
+    a {
+        text-decoration: none;
+    }
+    a, h1, h2 {
+        color: #248BB8;
+    }
+    a:hover, a:focus {
+        color: #DD291E;
+    }
+    h1, h2, th {
+        font-family: "Glegoo",Georgia,Times New Roman,serif;
+        text-transform: uppercase;
+    }
+    h1, h2 {
+        font-weight: 100;
+    }
+    </style>
+</head>
+<body>
+<h1>Sword Packaging for EASY Sword-v1</h1>
+<p>
+    <a href="http://easy.dans.knaw.nl/">EASY</a> supports a
+    <a href="https://easy.dans.knaw.nl/sword/servicedocument">Sword-v1</a> interface for deposits of
+    datasets. The zip file for a deposit should contain:
```

Comment:
use "must" instead of "should". "Should" implies there may be exceptions to the rule.